wwwroot/data/regionids/region_map-FID_LGA_2013_AUST_STE_NAME11.json

```diff
@@ -0,0 +1 @@
estern Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Western Australia","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Tasmania","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Northern Territory","Australian Capital Territory","Australian Capital Territory","Other Territories","Other Territories"]}
```

Comment:
>Should the first line read "STE_NAME13" instead of "STE_NAME11"?

It refers to the column/attribute, which for that layer really is STE_NAME11. I don't fully understand the ABS's processes, but I think it's along the lines that major datasets are only revised every 5 years or whatever, so you can have this intermediate dataset where the LGA properties are 2013, and the STE properties are 2011.

There is an extraneous carriage return, true.